src/Backend/One/OneToolchain.ts

```diff
@@ -34,6 +35,10 @@ class OneDebianToolchain extends DebianToolchain {
   run(cfg: string): Command {
     const configs = vscode.workspace.getConfiguration();
     const value = configs.get("one.toolchain.githubToken", "");
+    if (!value) {
```

Comment:
As I told in offline, it is a function that is not necessary for the public users. It would be good to give information to the user with the issue.

Reply:
I will give it as a balloon information :-D How about that?